hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -692,6 +711,13 @@ public void updateRpc(MethodDescriptor method, Message param, CallStats stats, T
     // this implementation is tied directly to protobuf implementation details. would be better
     // if we could dispatch based on something static, ie, request Message type.
     if (method.getService() == ClientService.getDescriptor()) {
+      if (
+        "Get".equals(method.getName()) || "Mutate".equals(method.getName())
+          || "Scan".equals(method.getName()) || "Multi".equals(method.getName())
+      ) {
+        updateTableMetric(methodName.toString(), tableName, stats, e);
```

Comment:
do we still need this, or can we simply add the call to `updateTableMetric` in the appropriate switch state cases below? Just to avoid unnecessary complexity and equality checks

Reply:
done.